src/opencage.rs

```diff
@@ -608,8 +606,20 @@ pub struct Status {
 #[derive(Debug, Serialize, Deserialize)]
 pub struct Timestamp {
     pub created_http: String,
-    #[serde(deserialize_with = "from_ts")]
-    pub created_unix: NaiveDateTime,
+    pub created_unix: UnixTime,
+}
+
+/// Primitive unix timestamp
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+pub struct UnixTime(i64);
+
+impl UnixTime {
+    pub const fn into_seconds(self) -> i64 {
```

Comment:
as_seconds() would be more appropriate according to the [Rust API Guidelines](https://rust-lang.github.io/api-guidelines/naming.html).

Reply:
done :)